cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorMockTest.java

```diff
@@ -464,7 +466,7 @@ public void testUpdateKafkaWithChangedPersistentVolume(TestContext context) {
         Kafka changedClusterCm = new KafkaBuilder(cluster).editSpec().editKafka()
                 .withNewPersistentClaimStorageStorage()
                     .withStorageClass(changedClass)
-                    .withSize("100Gi")
+                    .withSize("123")
```

Comment:
Why did you need to change this?

Reply:
Because the parameters used for it has "123" as size for storage. This test had to be fail before my changes but it wasn't failing because checking that the size cannot be changes wasn't working so we were "lucky" not having this test failing.